src/main/java/ch/uzh/ifi/hase/soprafs23/service/GameService.java

```diff
@@ -295,4 +295,16 @@ public Card getDrawnCard(int accessCode) {
     public List<Game> getAllGames() {
         return gameRepository.findAll();
     }
+
+    public void shuffleCards(int accessCode) {
+        Game existingGame = gameRepository.findByAccessCode(accessCode);
+        if (existingGame == null) {
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND, "Game with accessCode " + accessCode + " does not exist");
```

Comment:
use the const gameWithDode and doesNotExist instead of the "Game with accessCode"... to avaoid a codesmell